app/admin/layout.tsx

```diff
@@ -0,0 +1,18 @@
+"use client";
+
+import { Toaster } from "react-hot-toast";
+
+export default function SignInLayout({ children }: { children: React.ReactNode }) {
+  return (
+    <>
+      <main>{children}</main>
+      <Toaster position="top-center"
```

Comment:
```suggestion
      <Toaster position="top-right"
```
This aligns with the layout of website wdyt
https://github.com/pbdsce/PB_Website/blob/fb2c89133344fb3e2c0edcbbf629794fe9d801bd/app/(default)/layout.tsx#L33

Reply:
> This aligns with the layout of website wdyt
> 
> https://github.com/pbdsce/PB_Website/blob/fb2c89133344fb3e2c0edcbbf629794fe9d801bd/app/(default)/layout.tsx#L33

@mohit-nagaraj , Sure bhaiya, i can do that....

Just check it out once bhaiya, if it looks good then i will change that line

![Screenshot 2025-06-03 122438](https://github.com/user-attachments/assets/7d9d5517-03c9-4181-84c9-eaa9c900de30)

Reply:
Great!! Pls do push it